workflow/rocoto/workflow_tasks.py

```diff
@@ -1020,6 +1034,9 @@ def arch(self):
         if self.app_config.do_vrfy:
             dep_dict = {'type': 'task', 'name': f'{self.cdump}vrfy'}
             deps.append(rocoto.add_dependency(dep_dict))
+        if self.app_config.do_fit2obs and self.cdump in ['gdas']:
+            dep_dict = {'type': 'task', 'name': f'{self.cdump}fit2obs'}
+            deps.append(rocoto.add_dependency(dep_dict))
```

Comment:
Does metp actually depend on fit2obs?

Reply:
This is me adding fit2obs to the archive job dependency, not metp.

Reply:
d'oh, missed the code break where GH skipped lines